tsmo/README.md

```diff
@@ -0,0 +1,22 @@
+This directory contains several scripts that can be used to extract relevant data from the various kafka topics and generate
+the required plots to validate Carma-Streets UC3 functionality. The user should clone this repo and switch to the UC3 Analysis 
+branch. Inside of the "tsmo" directory, create a "data" directory with a set of subdirectories that are called out in the "constants.py"
```

Comment:
I guess we could remove the "The user should clone this repo and switch to the UC3 Analysis 
branch." part